app/views/index.ejs

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title><%= title %></title>
+    </head>
+    <body>
+        <h1>OG YOGA</h1>
```

Comment:
nit: in index.js, you have a dash "OG-YOGA", but here you do not have a dash. Maybe you should pick one to be consistent.

Reply:
Very good comment, Consistency is super important